### PR TITLE
Cancel ococo orders issue fix

### PR DESCRIPTION
**Pull request by avsek477**
While cancelling the oco orders / stopping the ococo algo, the auxillary order set by this algo (after the initial order is fulfilled) is not cancelled. This PR uses cancel multiple order by gid feature to resolve this issue.

Related PR: https://github.com/bitfinexcom/bfx-api-node-core/pull/22

Task: https://app.asana.com/0/1125859137800433/1200407232782235

### CODE REVIEW

File: lib/host/gen_helpers.js [hunk 1]
```diff
@@ -261,6 +263,43 @@ const genHelpers = (state = {}, adapter) => {
...
+      const cancelledActiveOrders = Object.values(orders).reduce((acc, o) => {
+        if (o.id && !excludeOrderStatusRegex.test(o.status)) {
+          acc[o.cid + ''] = o
+        }
+        return acc
+      }, {})
+
+      const otherOrders = _omit(state.orders, Object.keys(cancelledActiveOrders))
+
+      adapter.cancelOrdersByGid(connection, gid).then(() => { // eslint-disable-line
```
Comment (vitaliystoliarovcc) [inline lib/host/gen_helpers.js:282]: shouldn't be `await` here?

File: lib/host/gen_helpers.js [hunk 2]
```diff
@@ -261,6 +263,43 @@ const genHelpers = (state = {}, adapter) => {
...
 
+    cancelOrdersByGid: async (state = {}, gid) => {
+      logHelperCall('cancelOrderByGid: %s', gid)
+
+      const { connection, orders = {} } = state
+
+      const excludeOrderStatusRegex = new RegExp(excludeOrderStatus.join('|'))
+
+      const cancelledActiveOrders = Object.values(orders).reduce((acc, o) => {
+        if (o.id && !excludeOrderStatusRegex.test(o.status)) {
```
Comment (vitaliystoliarovcc) [inline lib/host/gen_helpers.js:274]: RegExp can be replaced with includes (of course, if the status is enum type)

**Comment by vitaliystoliarovcc** [comment]
Tested, everything works fine

File: lib/host/gen_helpers.js (hunk 1 above)
Comment (avsek477) [reply lib/host/gen_helpers.js]: fixed.

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**